TurkishFilter/sections/css_extended.txt

```diff
@@ -6,6 +6,7 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+l.ue2nx.xyz#?#pjsdiv[style*="font-size:"]:not([style*="pointer-events:"]):has(> pjsdiv[style] > pjsdiv[style*="font-family:"][style*="white-space:"])
```

Comment:
Please add the domain to the existing rule:
`#?##oframeplayerjs > pjsdiv:has(> pjsdiv[style] > pjsdiv:contains(FREESPİN))`